modules/serverless-rn-1-3-0.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-3-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.3.0
+
+[id="new-features-1-3-0_{context}"]
+== New features
+* {ServerlessProductName} has been updated to use Knative Serving 0.10.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.10.0.
+
```

Comment:
Any reason why this doesn't include HTTPs and/or the bug fixes we've listed in the Changelog?